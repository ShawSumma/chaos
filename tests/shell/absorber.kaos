asdas
print "hey"
a = 5
num a = 5
print a
num a = 3
print a
exit
