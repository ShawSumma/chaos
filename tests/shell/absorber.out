Chaos Error (most recent call last):
File: "~/chaos/__interactive__.kaos", line 1, in <module>
asdas
Undefined variable: asdas
Absorbed by Interactive Shell
hey
Chaos Error (most recent call last):
File: "~/chaos/__interactive__.kaos", line 3, in <module>
a = 5
Undefined variable: a
Absorbed by Interactive Shell
5
Chaos Error (most recent call last):
File: "~/chaos/__interactive__.kaos", line 6, in <module>
num a = 3
The variable name 'a' is already defined!
Absorbed by Interactive Shell
5
print b
Chaos Error (most recent call last):
File: "~/chaos/__interactive__.kaos", line 12, in <module>
b()
Undefined function: b
Absorbed by Interactive Shell
print b
